build/configs/rtl8720e/hello/defconfig

```diff
@@ -269,7 +269,20 @@ CONFIG_AUTOMOUNT_USERFS=y
 #
 # SE Selection
 #
-# CONFIG_SE is not set
+CONFIG_SE=y
+CONFIG_DEBUG_SECURE_ELEMENT_ERROR=y
+# CONFIG_SE_SSS is not set
+# CONFIG_SE_KONAI is not set
+# CONFIG_SE_VIRTUAL is not set
+CONFIG_SE_AMEBA=y
+CONFIG_HW_RNG=y
+CONFIG_HW_DH_PARAM=y
+CONFIG_HW_ECDH_PARAM=y
+CONFIG_HW_RSA_VERIFICATION=y
+CONFIG_HW_ECDSA_VERIFICATION=y
+CONFIG_HW_RSA_ENC=y
+CONFIG_HW_SE_STORAGE=y
+CONFIG_SE_SECURE_CONTEXT_SIZE=4096
```

Comment:
It is set as 6144 in TP1x. Is it right that this is different with TP1x?

Reply:
@ZhenBei-Sin As I remember, it was increased to 6144 to support one more additional algorithm. Please check it.

Reply:
https://github.com/Samsung/TizenRT/commit/6f2f189738c127935b440c6814b39ce97b7f292f
This is for ED25519 support. @j11591 @sangwon03 @an4967 @taehwan-hwang Could you confirm ED25519 support is necessary?

Reply:
Yes, 6144 is better, as the default has been changed to 6144
We will change it to 6144